src/pivot.coffee

```diff
@@ -572,7 +572,11 @@ callWithJQuery ($) ->
         getAggregator: (rowKey, colKey, forceDefaultTotalsAgg = false) =>
             flatRowKey = rowKey.join(FLAT_KEY_DELIM)
             flatColKey = colKey.join(FLAT_KEY_DELIM)
+
             getMetaAgg = @opts.totalsMetaAggregator and not forceDefaultTotalsAgg
+            # Don't use meta aggregators if they are not populated. This can occur when using only rows or only columns
+            getMetaAgg = getMetaAgg and (Object.keys(@metaAggRowTotals).length > 0 and
```

Comment:
Here you don't need the parens since you're not combining ANDs and ORs.

Also might want to define a separate variable above called `metaAggsPopulated` and then just include that in the `getMetaAgg =` line?

Reply:
Alternatively you could update forceDefaultTotalsAgg to true.

BTW can we rename `getMetAgg` to `useMetaAgg`? `get` implies it's a function that returns something, when in reality it's a boolean flag.